apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts

```diff
@@ -0,0 +1,63 @@
+import test from "../../fixtures/common";
+import { Account } from "tests/enum/Account";
+import { AppInfos } from "tests/enum/AppInfos";
+import { setExchangeDependencies } from "@ledgerhq/live-common/e2e/speculos";
+import { Fee } from "tests/enum/Fee";
+import { Swap } from "tests/models/Swap";
+import { Providers, Rates } from "tests/enum/Swap";
+
+const swaps = [
+  {
+    swap: new Swap(Account.ETH_1, Account.BTC_1, "0.022", Fee.MEDIUM),
+  },
+];
+
+const app: AppInfos = AppInfos.EXCHANGE;
+
+for (const { swap } of swaps) {
+  test.beforeAll(async () => {
+    process.env.SWAP_DISABLE_APPS_INSTALL = "true";
```

Comment:
I don't see a restoration of previous values, is it defined for the test and not used elsewhere?

Reply:
it isn't used elsewhere
added an `afterAll` to delete those values anyway

Reply:
added an `afterAll` to delete values
they ain't used elsewhere within our tests